doc/sphinxman/source/external.rst

```diff
@@ -229,6 +267,105 @@ to explicitly control the number of threads used for integral formation. Setting
 this variable to 0 (the default) uses the number of threads specified by the
 :py:func:`~p4util.util.set_num_threads` Psithon method or the default environmental variables.
 
+.. index:: PBS queueing system, threading
+.. _`sec:PBS`:
+
+PBS job file
+============
+
+To run a |PSIfour| job on a PBS queueing system, you need to properly set up
+all necessary variables in the PBS job file. Below is a minimal example of
+a PBS job file for a threaded job, and a short explanation for each section. ::
+
+    #!/bin/tcsh
+    #PBS -j oe 
+    #PBS -l pmem=2120mb
+    #PBS -N jobname
+    #PBS -V
+    
+    
+    setenv OMP_NUM_THREADS 4
+    setenv MKL_NUM_THREADS 4
+    cd $PBS_O_WORKDIR
+    setenv myscratch /scratch/user/psi4.$PBS_JOBID
+    
+    foreach i (`sort $PBS_NODEFILE | uniq`)
+        echo "Creating scratch directory " $myscratch " on " $i
+        ssh $i rm -rf $myscratch
+        ssh $i mkdir -p $myscratch
+    end
+
+    setenv PSI_SCRATCH $myscratch
+    setenv PSIPATH /path/to/external/modules:${PSIPATH}
+    setenv PSIPATH /path/to/python/modules:${PSIPATH}
+    setenv PSIDATADIR /psi/install/directory/share
+    /psi/install/directory/bin/psi4 -i input.in -o input.out
+    
+    foreach i (`sort $PBS_NODEFILE | uniq`)
+        echo "Removing scratch directory " $myscratch " on " $i
+        ssh $i rm -rf $myscratch
+    end
+
+The top section features PBS-specific commands. These depend on the 
+specific characteristics of your PBS queuing system but they may include: ::
+
+    #!/bin/tcsh
+    #PBS -j oe 
+    #PBS -l pmem=2120mb
+    #PBS -N jobname
+    #PBS -V
+    
+The ``PBS -j oe`` option instructs PBS to write any output or error message
+from the queuing system in dedicated files. ``PBS -l pmem=2120mb`` requests 
+2120 MB of memory for each thread on the node. The total memory requested for 
+the job by PBS should generally be slightly greater than what indicated 
+with the |adc__memory| keyword in the input file.
```

Comment:
Correct, "memory" is not a keyword but a command. It lacks "set" but gains units. What you're actually calling is http://psicode.org/psi4manual/master/autodoc_driver.html#p4util.util.set_memory .